src/app/elastic/search.py

```diff
@@ -39,8 +40,37 @@ def node_filter(self, resource_type: ResourceType, node_id: uuid.UUID) -> Search
         search = self.type_filter(resource_type=resource_type)
 
         if resource_type is ResourceType.COLLECTION:
-            return search.filter(qterm(qfield=ElasticResourceAttribute.PATH, value=node_id))
-        return search.filter(qterm(qfield=ElasticResourceAttribute.COLLECTION_PATH, value=node_id))
+            return search.filter(
+                qterm(qfield=ElasticResourceAttribute.PATH, value=node_id)
+            )
+        return search.filter(
+            qterm(qfield=ElasticResourceAttribute.COLLECTION_PATH, value=node_id)
+        )
+
+    def filter_serial_objects_out(self):
+        return self.filter(
+            Q("bool", **{"must_not": [{"term": {"aspects": "ccm:io_childobject"}}]})
+        )
+
+    def filter_text_only_objects(self):
```

Comment:
text_only_filter() to be in line with the other ..._filters?

Reply:
done